File: components_page/components/list_group/horizontal.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

list_group <- htmlDiv(
  list(
    dbcListGroup(
      list(
        dbcListGroupItem("Item 1"),
        dbcListGroupItem("Item 2"),
        dbcListGroupItem("Item 3")
      ),
      horizontal = TRUE,
      class_name = "mb-2",
    ),
    dbcListGroup(
      list(
        dbcListGroupItem("Item 1"),
        dbcListGroupItem("Item 2"),
        dbcListGroupItem("Item 3")
      ),
      horizontal = "lg",
    )
  )
)

File: components_page/components/list_group/horizontal.jl
using DashBootstrapComponents

list_group = html_div([
    dbc_listgroup(
        [
            dbc_listgroupitem("Item 1"),
            dbc_listgroupitem("Item 2"),
            dbc_listgroupitem("Item 3"),
        ],
        horizontal = true,
        className = "mb-2",
    ),
    dbc_listgroup(
        [
            dbc_listgroupitem("Item 1"),
            dbc_listgroupitem("Item 2"),
            dbc_listgroupitem("Item 3"),
        ],
        horizontal = "lg",
    ),
]);
